app/views/users/index.html.haml

```diff
@@ -15,7 +15,7 @@
     %tbody
       - @users.each do |user|
         %tr
-          %td= user.email
+          %td= link_to user.email, user
           %td
             %small= #{time_ago_in_words(user.created_at)} ago
```

Comment:
Please change this line to:
```
%small= "#{time_ago_in_words(user.created_at)} ago"
```
(add double quotes around the string - not sure why this stopped working (nothing to do with your PR)).